src/cr/cube/matrix/assembler.py

```diff
@@ -710,6 +710,23 @@ def _rows_dimension(self):
 class _ColumnOrderHelper(_BaseOrderHelper):
     """Encapsulates the complexity of the various kinds of column ordering."""
 
+    @lazyproperty
+    def _order(self):
+        """tuple of signed int idx for each column of measure matrix.
+
+        Negative values represent inserted-vector locations. Returned sequence reflects
+        insertion, hiding, pruning, and ordering transforms specified in the
+        rows-dimension.
+        """
+        CollatorCls = (
+            ExplicitOrderCollator
+            if self._columns_dimension.collation_method == CM.EXPLICIT_ORDER
+            else PayloadOrderCollator
+        )
+        return CollatorCls.display_order(
```

Comment:
is the signature going to change for future collators? maybe worth noting if it is...

Reply:
I expect it to remain the same for these two "anchored" orders (payload and explicit). All other collators are handled by their own distinct class, so it can handle variation in the call signature.